apps/site/src/lib/sanity/client.ts

```diff
@@ -0,0 +1,12 @@
+import { createClient } from 'next-sanity'
```

Comment:
Nitpick: Since other TS files in the site have semicolons at the ends of lines, this file should have them too.